### DISCUSSION TITLE
Capstone ML - Error Knitting

### DISCUSSION THREAD

**Discussion by namiraprita** [teamalgoritma/community#354] [Q&A] [answered]
Hi Team Algoritma, 

Saat saya mencoba untuk knit document to HTML selalu terjadi error seperti ini:
![Screen Shot 2022-09-26 at 07 50 22](https://user-images.githubusercontent.com/107253301/192269304-508c023f-d7a1-460d-991e-1a4c04395377.png)

Dari sisi directory, saya sudah memastikan kalau semua filenya sudah ada:

![Screen Shot 2022-09-26 at 07 51 4
[Capstone.zip](https://github.com/teamalgoritma/community/files/9645930/Capstone.zip)
5](https://user-images.githubusercontent.com/107253301/192269472-c8e12ffc-944b-41f4-accd-dccebcfef7e9.png)

Apakah ada solusinya dari ini? Attached rmd file dan file2 lainnya ya. Terima kasih atas bantuannya

**Comment by fafilia** [comment, marked as answer]
Dear @namiraprita ,

Jika dilihat dari codenya sepertinya tidak ada masalah. Mungkin boleh dicoba bu untuk memindahkan filenya ke path yang lebih ringkas. Jika dilihat dari path yang terlampir, sepertinya Ibu menyimpan filenya cukup dalam di beberapa folder. Bisa jadi gagal knittingnya karena masalah tersebut karena setahu saya R memiliki limitasi dengan hal tersebut. Silahkan dicoba terlebih dahulu Bu.
